test/unit/api/wide/load/arithmetic/load.hpp

```diff
@@ -26,83 +26,236 @@ auto data_block()
+    {
+      TTS_EQUAL(T(eve::as_aligned<algt>(ref_const_ptr))                                    , ref);
 
-  T constructed(eve::as_aligned<algt>(ref_ptr));
-  TTS_EQUAL(constructed, ref);
+      if constexpr(EVE_CARDINAL == eve::expected_cardinal_v<EVE_VALUE>)
+        TTS_EQUAL(eve::load(eve::as_aligned<algt>(ref_const_ptr))                          , ref);
 
-  T loaded = eve::load(eve::as_aligned<algt>(ref_ptr), eve::as_<T>());
-  TTS_EQUAL(loaded, ref);
+      TTS_EQUAL((eve::load(eve::as_aligned<algt>(ref_const_ptr), eve::lane<EVE_CARDINAL>)) , ref);
+    }
+  }
+}
+
+TTS_CASE_TPL("Check conditional load from unaligned pointer for wide", EVE_TYPE)
+{
+  constexpr std::ptrdiff_t algt = eve::alignment_v<T>;
+  auto [data,idx] = data_block<EVE_VALUE, eve::fixed<EVE_CARDINAL>>();
+  auto* ref_ptr             = &data[idx];
+  auto const* ref_const_ptr = &data[idx];
+
+  TTS_WHEN("For some given relative masks")
+  {
+    // Conditional selectors
+    auto il = eve::ignore_last(EVE_CARDINAL/4);
+    auto i1 = eve::ignore_first(EVE_CARDINAL/4);
+    auto kf = eve::keep_first(EVE_CARDINAL/4);
+    auto kl = eve::keep_last(EVE_CARDINAL/4);
+    auto kb = eve::keep_between ( std::min(0L,EVE_CARDINAL)
+                                , std::max(0L,EVE_CARDINAL-2)
+                                );
+    auto ie = i1 && il;
+
+    // Conditional selectors' masks
+    auto ml   = il.mask( eve::as_<T>() );
+    auto m1   = i1.mask( eve::as_<T>() );
+    auto mkf  = kf.mask( eve::as_<T>() );
+    auto mkl  = kl.mask( eve::as_<T>() );
+    auto mkb  = kb.mask( eve::as_<T>() );
+    auto mie  = ie.mask( eve::as_<T>() );
+    auto mia  = eve::ignore_all.mask( eve::as_<T>() );
+
+    // Reference values
+    T full_ref(ref_ptr);
+    T ignore_last_ref   = full_ref & ml.mask();
+    T ignore_first_ref  = full_ref & m1.mask();
+    T keep_first_ref    = full_ref & mkf.mask();
+    T keep_last_ref     = full_ref & mkl.mask();
+    T keep_between_ref  = full_ref & mkb.mask();
+    T ignore_ext_ref    = full_ref & mie.mask();
+
+    // lanes value
+    auto lanes = eve::lane<EVE_CARDINAL>;
+
+    TTS_AND_THEN("load is applied on unaligned pointer for default cardinal")
+    {
+      if constexpr(EVE_CARDINAL == eve::expected_cardinal_v<EVE_VALUE>)
+      {
+      TTS_EQUAL(eve::load[eve::ignore_none](ref_ptr)              , full_ref        );
+      TTS_EQUAL((eve::load[il](ref_ptr) & ml.mask() )             , ignore_last_ref );
+      TTS_EQUAL((eve::load[i1](ref_ptr) & m1.mask() )             , ignore_first_ref);
+      TTS_EQUAL((eve::load[kf](ref_ptr) & mkf.mask())             , keep_first_ref  );
+      TTS_EQUAL((eve::load[kl](ref_ptr) & mkl.mask())             , keep_last_ref   );
+      TTS_EQUAL((eve::load[kb](ref_ptr) & mkb.mask())             , keep_between_ref);
+      TTS_EQUAL((eve::load[ie](ref_ptr) & mie.mask())             , ignore_ext_ref  );
+      TTS_EQUAL((eve::load[eve::ignore_all](ref_ptr) & mia.mask()), T{0}            );
```

Comment:
That's not guaranteed to be 0 probably, right?

I would also suggest to load from really less data then a register.

Finally - I am not sure it's OK to compare for full equality - again - we don't populate garbage values